pkg/runner/deployer/ssh.go

```diff
@@ -135,3 +147,54 @@ func (p *SSH) Run(cmdSegs []string, handler DeployerHandler) (err error) {
 
 	return nil
 }
+
+type FSStats struct {
+	Path       string `json:"path"`
+	BlockSize  uint64 `json:"block-size"`
```

Comment:
Since you're voluntarily defining this, I'd get in the habit of using camelCase for these keys.